tables/de-g0-core.uti

```diff
@@ -1,6 +1,7 @@
-# liblouis: German grade 0 braille
+# liblouis: German grade 0 braille (bidirectional)
```

Comment:
"bidirectional" -> "with indication of capitals"

Reply:
Would "detailed" be OK? Because it is with capitals and with detailed accents

Reply:
See my other comment.